test/object_store/test_basic.py

```diff
@@ -88,7 +91,7 @@ def managed_cluster(run_dir, ssl, s3_server):
     run.wait_for_services(pid, [check_with_cql(ip, ssl)])
     cluster = run.get_cql_cluster(ip)
     try:
-        yield cluster
+        yield Cluster(cluster)
```

Comment:
might want to update the comment couple lines above. it now reads

> it yields an instance of cassandra.Cluster referencing this cluster.

after this change, it is not accurate anymore.

Reply:
Good point. Fixed